consensus/polybft/signer/private_test.go

```diff
@@ -0,0 +1,23 @@
+package bls
+
+import (
+	"fmt"
+	"testing"
+)
+
+func Test_PrivateMarshal(t *testing.T) {
+	// blsKey, err := GenerateBlsKey() // structure which holds private/public key pair
+	// require.NoError(t, err)
+	//
+	// // marshal public key
+	// privateKeyMarshalled, err := blsKey.MarshalJSON()
+	// require.NoError(t, err)
+	// // recover private and public key
+	// blsKeyUnmarshalled, err := UnmarshalPrivateKey(privateKeyMarshalled)
+	// require.NoError(t, err)
+	//
+	// assert.Equal(t, blsKey, blsKeyUnmarshalled)
+	var b []byte
+
+	fmt.Println(len(b))
```

Comment:
Is it a test?

Reply:
You have the point, not sure why this is currently commented. Let me try fix it.